<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AxisLimit</class>
 <widget class="QWidget" name="AxisLimit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="gpLimit">
     <property name="title">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="spinU1">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Base</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="spinL0">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <double>-170.000000000000000</double>
        </property>
        <property name="maximum">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="spinU2">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="spinL1">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <double>-80.000000000000000</double>
        </property>
        <property name="maximum">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Wrist</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="spinL3">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="spinU3">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Elbow</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="spinL2">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <double>-50.000000000000000</double>
        </property>
        <property name="maximum">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Shoulder</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="spinU0">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <double>170.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
